designs/administrator/future-openshift/cluster-setup/index.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.X
+---
+
+# Cluster Setup
+
+## First time use
+![](img/setup-hint.png)
+When the user first signs into OpenShift after installation they are presented with a hint block that directs them to complete the cluster setup steps. Clicking on the action button in the hint should open the notification drawer.
```

Comment:
Would this block note only appear on the first login to the cluster, and not subsequent ones? Is there any other way to dismiss it other than clicking the view steps link, perhaps if the notification drawer was opened manually or maybe including a dedicated dismiss action?

Reply:
Updated this with more details